StarVMC/StarAgmlLib/AgPosition.cxx

```diff
@@ -35,7 +36,12 @@ void AgMLDbMatrix::operator()( const char* table, const int row, double matrix[4
   TString tablename = table;
   tablename.ReplaceAll(" ","");
 
-  TTable *ttable = (TTable *)chain-> GetDataBase( tablename );
+#if ROOT_VERSION_CODE < ROOT_VERSION(6,00,0) 
+  TTable* ttable = (TTable *)StMaker::GetChain() -> GetDataBase( tablename );
+#else
+  TTable* ttable = 0;
+#endif
```

Comment:
This is probably the most controversial change in this PR since you deliberately break the code for ROOT6. If we leave it as is it maybe easier to see the problem when use it with ROOT6

Reply:
I believe I did this anticipating problems with GetDataBase( tablename) under root 6.  

GetDataBase loads / unloads macros which look like this:
https://github.com/star-bnl/star-sw/blob/main/StarDb/Geometry/pxl/pxlLadderOnSectorMisalign.20131210.000001.C
https://github.com/star-bnl/star-sw/blob/main/StarDb/Geometry/ist/istLadderOnIstMisalign.20131210.000001.C

They all have the same function defined...  CINT handles this reasonably well, as the interpreter can just drop the function.
When I started geant4star, it was an open question whether or not ROOT6 would handle this gracefully.  Once we are ready to use ROOT 6 in production, we can restore this line.

Reply:
Addressed in 1bc5b0c4fd923ab1fdcd2c55ad600c42ae411f28